database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,13 +14,15 @@
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
+            $table->increments('id');
```

Comment:
I know you probably did this intentionally but as an FYI, `id()` is an alias for `bigIncrements()`